flake.nix

```diff
@@ -410,6 +416,11 @@
               postUnpack = "sourceRoot=$sourceRoot/perl";
             });
 
+            passthru.python-bindings = final.callPackage ./python {
+              inherit self system;
```

Comment:
```suggestion
```

These shouldn't be necessary. Only `example-buildPythonApplication` uses `system`, but can't `callPackage` put `pkgs` into that example expression? `self` is unused.